src/helpers/functionsTodos.tsx

```diff
@@ -0,0 +1,24 @@
+import { Todo } from '../types/Todo';
+import { Filter } from '../types/Filter';
+
+export const filteredTodos = (todos: Todo[], filter: Filter) => (
+  todos.filter(({ completed }) => {
+    switch (filter) {
+      case Filter.All:
+        return true;
+
+      case Filter.Active:
+        return !completed;
+
+      case Filter.Completed:
+        return completed;
+
+      default:
+        return 0;
+    }
+  })
+);
+
+export const activeTodosCount = (todos: Todo[]) => (
+  todos.filter(todo => !todo.completed)
```

Comment:
if you named this function like `activeTodosCount`, other developers expect this function to return a number
```suggestion
  todos.filter(todo => !todo.completed).length
```